MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: post
+title: Getting started with .NET MAUI Picker control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI Picker (SfPicker) control and its basic features.
```

Comment:
description: Learn about getting started with Syncfusion .NET MAUI Picker (SfPicker) control and its basic features.

Reply:
Content modified.